packages/runner-shared/src/header/index.tsx

```diff
@@ -262,6 +262,8 @@ export class Header extends Component<HeaderProps> {
 
     if (!this._studioNeedsUrl) return
 
+    // TODO: studio/studio-recorder.js should be converted to ts. And add proper type.
```

Comment:
Can we log an issue for this work?

Reply:
I added more comment about it.

Reply:
I was thinking we'd log an issue to update studio/studio-recorder.js to be converted to typescript, not necessarily reference this PR. We should be able to find this PR reference in the git history.

Reply:
```js
// Note: changes in TypeScript 4.4.4 made the code below fail the type check.
// ts interprets visitUrl below as (url: null | undefined) => never
// TODO: studio/studio-recorder.js should be converted to ts. And add proper type.
// @ts-ignore
```

I left why this `studio/studio-recorder.js` should be converted to ts.